src/pages/das-api/guides/get-wallet-tokens.md

```
@@ -0,0 +1,169 @@
+  walletTokens.items.forEach((token) => {
+    console.log(`Token: ${token.id}`);
+    console.log(`Interface: ${token.interface}`);
+    console.log(`Name: ${token.content.metadata?.name || "Unknown"}`);
+  });
+})();
+```
+{% /totem-accordion %}
+{% totem-accordion title="JavaScript Example" %}
+```javascript
+(async () => {
+  const response = await fetch("<ENDPOINT>", {
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)